app/components/list/model.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const striptags = require('striptags'),
+  { has, isFieldEmpty } = require('../../services/universal/utils'),
+  { render } = require('../../services/universal/styles'),
+  { toSmartText } = require('../../services/universal/sanitize');
+
+module.exports.save = function (uri, data) {
+  if (has(data.items)) {
+    data.items.forEach((item) => {
+      item.text = toSmartText(striptags(item.text, ['strong', 'em', 's', 'a', 'span']));
```

Comment:
Maybe we can put this in a variable in case we want to add more tags in the future
```suggestion
      item.text = toSmartText(striptags(item.text, allowedTags));
```